tests/performance/hashed_dictionary.xml

```diff
@@ -48,91 +76,70 @@
             value_string_nullable Nullable(String)
         )
         PRIMARY KEY id, id_key
-        SOURCE(CLICKHOUSE(DB 'default' TABLE 'complex_key_hashed_dictionary_source_table'))
-        LAYOUT(COMPLEX_KEY_HASHED())
-        LIFETIME(MIN 0 MAX 1000);
+        SOURCE(CLICKHOUSE(TABLE 'complex_key_dictionary_source_table'))
+        LAYOUT(COMPLEX_KEY_{layout_suffix}())
+        LIFETIME(0)
     </create_query>
 
     <fill_query>
-        INSERT INTO simple_key_hashed_dictionary_source_table
+        INSERT INTO simple_key_dictionary_source_table
         SELECT number, number, toString(number), toDecimal64(number, 8), toString(number)
         FROM system.numbers
-        LIMIT 5000000;
+        LIMIT '{layout_suffix}' = 'HASHED' ? 500000 : 50000
     </fill_query>
 
     <fill_query>
-        INSERT INTO complex_key_hashed_dictionary_source_table
+        INSERT INTO complex_key_dictionary_source_table
         SELECT number, toString(number), number, toString(number), toDecimal64(number, 8), toString(number)
         FROM system.numbers
-        LIMIT 5000000;
+        LIMIT '{layout_suffix}' = 'HASHED' ? 500000 : 50000
```

Comment:
Fill queries are executed before main test queries.
So it will insert into complex_key_dictionary_source_table 2 times first 500000 with records second with 50000 records. Then during query dictionary will always work with table of 550000.

Reply:
@nickitat that's true.

You can simply copy-paste everything instead.